app/views/api/v1/users/_info.json.jbuilder

```diff
@@ -1,3 +1,3 @@
 json.id       user.id
 json.name     user.full_name
-json.gender   user.gender
+json.email    user.email
```

Comment:
why was removed gender?

Reply:
i thought it was not particularly important